pkg/ebpf/events_processor.go

```diff
@@ -333,6 +333,12 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 			return fmt.Errorf("error parsing cgroup_mkdir args: %w", err)
 		}
 		t.containers.CgroupRemove(cgroupId, hId)
+	case DebugfsCreateDir:
+		path, err := getEventArgStringVal(event, "path")
+		if err != nil {
+			return fmt.Errorf("error parsing debufs_create_dir args: %w", err)
+		}
+		event.Args[1].Value = fmt.Sprintf("/sys/kernel/debug/%s", path)
```

Comment:
Could you re-check this ? Based on my test in `debugfs_create_file`:

```
"args":[{"name":"file_name","type":"const char*","value":"testing123"},{"name":"path","type":"const char*","value":"/usb"}
```

You might get "//dirname" here. Also, you're assuming debugfs is always mounted in /sys/kernel/debug (not sure that is the case everywhere, needs checking.

Reply:
As this event is already called "debugfs_create_file", I don't think that this code is even needed

Reply:
I think that if it is possible to pass the full path to the file instead of only its path inside the FS it would be much more user friendly.
It will also match the way the `path` argument is used in other events.